### DISCUSSION TITLE
规则附加-第三方规则集附加-点添加按钮出错

### DISCUSSION THREAD

**Discussion by Buffnada** [vernesong/OpenClash#3094] [Q&A]
原本的表格结构 比如是如下
配置文件.yaml    规则集  direct ....
点击添加按钮后 啥操作没有表格结构
配置文件.yaml    规则集，规则集  direct ....
规则集自动添加重复自己一份  
如果这时候保存再点添加  规则集变成三个

45.100 有问题 上版本45.97也有问题  再上上次版本不是这样的

**Comment by vernesong** [comment]
dev已经修复
